src/Umbraco.Core/PropertyEditors/ValueConverters/DropdownListValueConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco.Core.PropertyEditors.ValueConverters
+{
+    public class DropdownListValueConverter : PropertyValueConverterBase, IPropertyValueConverterMeta
+    {
+        public override bool IsConverter(PublishedPropertyType propertyType)
+        {
+            return propertyType.PropertyEditorAlias.InvariantEquals(Constants.PropertyEditors.DropDownListAlias);
```

Comment:
A few of these are missing the if (UmbracoConfig.For.UmbracoSettings().Content.EnablePropertyValueConverters) check